manifests/qualys.pp

```diff
@@ -101,19 +101,30 @@
       uid            => $uid,
     }
 
-    if ( ! empty($subgid_file) and ! empty($subuid_file) ) {
-      # CLEAN UP qualys subuid/subgid FILE ENTRIES
-      File_line {
-        ensure            => 'absent',
-        match_for_absence => true,
-      }
-      file_line { "remove ${group} group from ${subgid_file}":
-        path  => $subgid_file,
-        match => "^${group}:*",
+    # SUBORDINATE MIN/MAX - NEED TO BE LESS/GREATER THAN /etc/login.defs OR OS DEFAULTS
+    $sub_gid_min = 0                   # RHEL DEFAULT = 100000
+    $sub_gid_max = 999999999999999999  # RHEL DEFAULT = 600100000
+    $sub_uid_min = 0                   # RHEL DEFAULT = 100000
+    $sub_uid_max = 999999999999999999  # RHEL DEFAULT = 600100000
+
+    if ( ! empty($subgid_file) ) {
+      exec { "remove ${group} user from ${subgid_file}":
+        command => "usermod --del-subgids ${sub_gid_min}-${sub_gid_max} ${group}",
+        onlyif  => "grep ${group} ${subgid_file}",
```

Comment:
I suggest using `egrep "^${group}:" ...` to avoid a mishap in the (admittedly unlikely) case that some other legit group whose name contains "qualys" is in the file.

Similar for the $user check below.